source/Features.rst

```diff
@@ -589,8 +604,19 @@ Choose the option like “Share”. Through this you can  share the details to o
 
 .. image:: images/testingevent4.png
 
+**3. Unshare Action**
+
+Choose the option like "Unshare". Through this you can unshare the details from the earlier shared team member.  
+
+.. image:: images/unshare.png
+
+.. image:: images/unshare1.png
 
-**3. Terminate Action**
+**Note:** The "Unshare" option is available only when the bucket is shared with other researchers. The owner(i.e.,person who provisioned product) can do the share and unshare. 
+
+**Note:** If there are no researchers in the list you will see a message like **“No researchers are available. Please add a new researcher to share the s3 bucket.“**
+
+**4. Terminate Action**
 
 Choose an option like “Terminate”. Through this you can terminate the product
```

Comment:
Remove the word "like" from this sentence. The sentence should be "Choose the 'Terminate' option to de-provision the product"